lib/hako/schedulers/ecs.rb

```diff
@@ -81,6 +81,7 @@ def configure(options)
           end
         end
         @requires_compatibilities = options.fetch('requires_compatibilities', nil)
+        @runtime_platform = options.fetch('runtime_platform', nil)
```

Comment:
It seems `runtime_platform` is a struct.
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/ECS/Types/RuntimePlatform.html
Hako should omit unsupported fields. So `@runtime_platform` should be built with supported fields like `@ephemeral_storage`, `@capacity_provider_strategy` and `@network_configuration`.